Corona-Warn-App/src/deviceForTesters/res/layout/fragment_test_appconfig.xml

```diff
@@ -73,6 +73,25 @@
             </LinearLayout>
         </LinearLayout>
 
+        <LinearLayout
+            style="@style/card"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_margin="@dimen/spacing_tiny"
+            android:orientation="vertical">
+
+            <Switch
+                android:id="@+id/device_time_check_disabled_toggle"
+                style="@style/body1"
+                android:layout_width="match_parent"
+                android:layout_height="0dp"
+                android:layout_marginTop="@dimen/spacing_tiny"
+                android:layout_weight="1"
+                android:text="Disable device time check"
+                android:theme="@style/switchBase" />
+
+        </LinearLayout>
+
```

Comment:
can you just use `Switch` without `LinearLayout`?

Reply:
We could, but I'm sure there will be more switches in the future to toggle other app config features, the whole box is there to contain those switches. We could maybe add a title to it, but IMHO it is okay as it is.